packages/neon_framework/packages/neon_http_client/test/interceptors/authorization_throttling_interceptor_test.dart

```diff
@@ -48,8 +65,23 @@ void main() {
       final authorizedRequest = Request('GET', baseURL);
       authorizedRequest.headers['authorization'] = 'test';
       final authorizedResponse = StreamedResponse(
-        const Stream.empty(),
+        Stream.value(
+          utf8.encode(
+            json.encode(
+              {
+                'ocs': {
+                  'meta': {
+                    'statuscode': 997,
+                  },
+                },
+              },
+            ),
+          ),
+        ),
         401,
+        headers: {
+          'content-type': 'application/json; charset=utf-8',
+        },
```

Comment:
The stram and header generation can be extended into a helper method `buildStream` and `buildHeaders`.